docs/articles/documentation/guides/continuous-integration/github-actions-and-browserstack.md

```diff
@@ -0,0 +1,252 @@
+```
+
+{% endraw %}
+
+## Step 5 - Add a Step To Install TestCafe BrowserStack plugin
+
+In order to enable TestCafe to run in the BrowserStack environment, install our [browserstack plugin](https://github.com/DevExpress/testcafe-browser-provider-browserstack).
+
+{% raw %}
+
+```yml
+name: End-to-End Tests
+on: [push]
+env:
+        BROWSERSTACK_ACCESS_KEY: "${{ secrets.BROWSERSTACK_ACCESS_KEY }}"
+        BROWSERSTACK_USERNAME: "${{ secrets.BROWSERSTACK_USERNAME }}"
+jobs:
+  test:
+    name: Run TestCafe Tests
+    runs-on: windows-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v1
+      - name: Install TestCafe BrowserStack plugin
+        run: npm install testcafe-browser-provider-browserstack
+```
+
+{% endraw %}
+
+## Step 6 - Add a Step That Runs TestCafe
+
+Add the [Run TestCafe](https://github.com/DevExpress/testcafe-action) action. Use the [args](#args) parameter to provide TestCafe [command line arguments](../../reference/command-line-interface.md), including the BrowserStack configuration:
+
+{% raw %}
+
+```yml
+name: End-to-End Tests
+on: [push]
+env:
+        BROWSERSTACK_ACCESS_KEY: "${{ secrets.BROWSERSTACK_ACCESS_KEY }}"
+        BROWSERSTACK_USERNAME: "${{ secrets.BROWSERSTACK_USERNAME }}"
+jobs:
+  test:
+    name: Run TestCafe Tests
+    runs-on: windows-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v1
+      - name: Run tests
+        uses: DevExpress/testcafe-action@latest
+        with:
+            args: "'browserstack:chrome@84.0:Windows 10' tests"
+```
+
+{% endraw %}
+
+> Note the additional pair of single quotes- BrowserStack configuration string includes a space, which can be interpreted as a separator by the command line.
+
+**Note**: Full list of available BrowserStack configurations can be obtained through console.
+First, you have to provide your credentials by exporting them:
+
+**bash**
+
+```sh
+export BROWSERSTACK_USERNAME="<your_browserstack_username>"
+export BROWSERSTACK_ACCESS_KEY="<your_browserstack_acess_key>"
+```
+
+**PowerShell**
+
+```sh
+Set-Variable -Name "BROWSERSTACK_USERNAME" -Value "<your_browserstack_username>"
+Set-Variable -Name "BROWSERSTACK_ACCESS_KEY" -Value "<your_browserstack_acess_key>"
+```
+
+Afterwards, run in console:
+
+```sh
+testcafe -b browserstack
+```
+
+## Action Options
+
+### args
+
+TestCafe [command line arguments](../../reference/command-line-interface.md).
+
+```yml
+- uses: DevExpress/testcafe-action@latest
+  with:
+    args: "'browserstack:chrome@84.0:Windows 10' fixture.js -s takeOnFails=true -q -c 3"
+```
+
+### version
+
+*Optional*
+
+The TestCafe version to install.
+
+```yml
+- uses: DevExpress/testcafe-action@latest
+  with:
+    version: "1.6.0"
+    args: "'browserstack:chrome@84.0:Windows 10' tests"
+```
+
+Commence the build by commiting changes and pushing to your repository. You can see the results on `Actions` page in your repository.
```

Comment:
What is pushed to the repository? The changes?

Reply:
PUSH is a name of the action, so i believe it is valid to say "push to the repository". You just type "git push" in console, and everything that need to get pushed, does.

technically, commits are being pushed though

Reply:
```suggestion
To start the build, commit changes and push to the repository. You can see the results on the `Actions` page in your repository.
```